src/test/java/com/sfuentes/EmployeeTest.java

```diff
@@ -0,0 +1,84 @@
+package com.sfuentes;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class EmployeeTest {
+
+  @Test
+  void test_exceptionIncorrectAmountOfSales() {
+    double[] sales = {100.00, 100.00, 100.00, 100.00};
+
+    Exception exception = assertThrows(IllegalArgumentException.class, () -> {
+      Employee test = new Employee(1, sales);
+    });
+
+    String expectedMessage = "Incorrect amount of sales";
+    String actualMessage = exception.getMessage();
+
+    assertTrue(actualMessage.contains(expectedMessage));
```

Comment:
Try to not use variables for the assertion, if possible please use a literal value or a method call.

Reply:
fixed